### PR TITLE
852 update to pkgdown in the ci workflow generates new warnings and errors

### PR DESCRIPTION
**Pull request by Puzzled-Face**
This PR is currently draft to allow for testing of fixes to the new `pkgdown` errors and warnings.

### CODE REVIEW

File: vignettes/trial_analysis.Rmd [hunk 1]
```diff
@@ -620,7 +624,7 @@ fullSamples %>%
   )
 ```
 
-```{r, error=TRUE}
+```{r, fig.alt = "A visual representation of the posterior dose - toxicity curve.  Very closely spaced sloid lines in black and blue, represewnting the mean and median estimate of toxicity for each dose rise almost linearly from zero percent for a dose of zero to about 55% for a dose of 100.  Shading extends to each side of the two solid lines.  The transparency of the shading increases with distance from the solid lines.  The shading is funnel shaped, with a narrow mneck at a dose of 100 and a wider mouth at a dose of 100.  The shading represents the central 90%, 80% and 50% confidence intervals for the posterior mean estimate of toxicity at each dose."}
```
Comment (danielinteractive) [inline vignettes/trial_analysis.Rmd:627]:
```suggestion
```{r, fig.alt = "A visual representation of the posterior dose - toxicity curve.  Very closely spaced sloid lines in black and blue, representing the mean and median estimate of toxicity for each dose rise almost linearly from zero percent for a dose of zero to about 55% for a dose of 100.  Shading extends to each side of the two solid lines.  The transparency of the shading increases with distance from the solid lines.  The shading is funnel shaped, with a narrow neck at a dose of 100 and a wider mouth at a dose of 100.  The shading represents the central 90%, 80% and 50% confidence intervals for the posterior mean estimate of toxicity at each dose."}
```

**Review by danielinteractive — COMMENTED** [review]
Thanks @Puzzled-Face for this cleanup, yes, ok to resize plots as needed.

**Comment by danielinteractive** [comment]
Is this a quarto pipeline error, or is the error triggered by missing alt text? (I see an error message from `pkgdown`)

**Comment by Puzzled-Face** [comment]
@danielinteractive Dinakar appears to have fixed the `pkgdown` workflow and I have regressed the `knit_print` vignette from Quarto to RMarkdown.  Changes in other recent PRs appear to have superseded this one, so I believe it can either be closed or merged as you prefer.

**Comment by danielinteractive** [comment]
@Puzzled-Face I guess the `fig.alt` additions are still useful?

**Review by danielinteractive — APPROVED** [review]

**Comment by Puzzled-Face** [comment]
> @Puzzled-Face I guess the `fig.alt` additions are still useful?

I believe so, though I haven't temporarily taken them out to see if the warnings reappear.  I expect they would.

**Pull request merged**